### PR TITLE
Add tox to test/requirements.txt

### PR DESCRIPTION
**Pull request by eamanu**
To be accordint to the documentation, add tox into requirements.

Closes: #697

### CODE REVIEW

**Comment by noviluni** [comment]
Hi @eamanu! Changes look good! However, could you also delete the line 
```
$ flake8 dateparser tests
```

from the fifth point inside "Get Started!"?

It's not necessary anymore because the flake8 is automatically run when running the tests with tox (ref: https://github.com/scrapinghub/dateparser/pull/665).

Thanks for helping with keeping the docs updated. :)

**Comment by noviluni** [comment]
Hey @eamanu! I updated my last comment :point_up:, take care if you check it from the e-mail!

**Comment by eamanu** [comment]
Hi @noviluni great, I will prepare the changes.

**Comment by eamanu** [comment]
@noviluni  Ready

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -41,7 +41,11 @@ DateParser could always use more documentation, whether as part of the
 official DateParser docs, in docstrings, or even on the web in blog posts,
 articles, and such.
 
-After you make local changes to the documentation, build it with ``tox``::
+After you make local changes to the documentation, you should install ``tox``::
+
+    pip install tox
+
+Then, you will be able to build the project running::
```
Comment (Gallaecio) [inline CONTRIBUTING.rst:48]: While I don’t expect people to do it, this reads like you need to install `tox` every time.

What about adding a `.. note::` below that explains that, if you do not have `tox` installed, you need to install it first using `pip install tox`?

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -84,8 +88,6 @@ Ready to contribute? Here's how to set up `dateparser` for local development.
 
 5. When you're done making changes, check that your changes pass flake8 and the tests, including testing other Python versions with tox::
 
-    $ pip install -r tests/requirements.txt # install test dependencies
-    $ pip install -r scripts/requirements.txt # install script dependencies
     $ flake8 dateparser tests
```
Comment (Gallaecio) [inline CONTRIBUTING.rst:91]: This line can go as well, [`tox` also runs flake by default](https://github.com/scrapinghub/dateparser/blob/master/tox.ini#L2).

**Comment by noviluni** [comment]
Hi @eamanu, could you delete the mentioned line (`flake8 dateparser tests`) and fix the conflicts to us to be able to merge it to master?

**Comment by eamanu** [comment]
Hi @noviluni @Gallaecio Sorry for the delay, tomorrow I will make the changes.

**Comment by eamanu** [comment]
Ready for review @noviluni @Gallaecio. thanks! and sorrry for the delay

File: CONTRIBUTING.rst [hunk 3]
```diff
@@ -41,13 +41,20 @@ DateParser could always use more documentation, whether as part of the
...
 
     tox -e docs
 
+
 Then open ``.tox/docs/tmp/html/index.html`` in a web browser to see your local
 build of the documentation.
 
+.. note::
+
+    If you don't have `tox` installed, you need to install ir first using
```
Comment (noviluni) [inline CONTRIBUTING.rst:55]: there is a typo in this sentence

Comment (eamanu) [reply CONTRIBUTING.rst]: Sorry. Ready @noviluni

File: CONTRIBUTING.rst [hunk 4]
```diff
@@ -41,13 +41,20 @@ DateParser could always use more documentation, whether as part of the
...
     tox -e docs
 
+
 Then open ``.tox/docs/tmp/html/index.html`` in a web browser to see your local
 build of the documentation.
 
+.. note::
+
+    If you don't have `tox` installed, you need to install it first using
+    `pip install tox`
```
Comment (Gallaecio) [inline CONTRIBUTING.rst:56]:
```suggestion
    If you don't have ``tox`` installed, you need to install it first using
    ``pip install tox``.
```

**Pull request merged**